js_modules/dagster-ui/packages/ui-components/src/components/__stories__/Icon.stories.tsx

```diff
@@ -52,7 +52,7 @@ export const IconColors = () => {
 
   return (
     <Box flex={{gap: 6, wrap: 'wrap'}}>
-      {IconNames.map((name, idx) => (
+      {IconNames.sort().map((name, idx) => (
```

Comment:
```suggestion
      {[...IconNames].sort().map((name, idx) => (
```

since `sort` is mutative